.husky/pre-commit

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-cd mw-webapp
-pnpm lint-staged
+FRONTEND="mw-webapp/"
+SERVER="mw-server/"
+
+if git diff --name-only --cached | grep $FRONTEND;
+  then
+    echo "FRONTEND FOLDER CHANGED --> starting husky..."
+    cd mw-webapp
+    pnpm lint-staged
+    pnpm dlx concurrently \
+        "pnpm type-check" \
+        "pnpm test" \
+        "pnpm build-storybook" \
```

Comment:
let's move such scripts to the root level and let's avoid to use "cd" command in such scripts, I hope it will be convenient